hw/ip/sysrst_ctrl/doc/checklist.md

```diff
@@ -71,6 +71,14 @@ Security      | [SEC_RND_CNST][]        | N/A         |
 [SEC_CM_DOCUMENTED]:   {{<relref "/doc/project/checklist.md#sec_cm_documented" >}}
 [SEC_RND_CNST]:        {{<relref "/doc/project/checklist.md#sec_rnd_cnst" >}}
 
+### D2S
```

Comment:
Update to this file should not have showed up in this PR.

Reply:
Reverted this file